### PR TITLE
Display License in github

### PR DESCRIPTION
**Pull request by a-teammate**
see commit message for info.

### CODE REVIEW

**Comment by Croydon** [comment]
FYI: I want to change a few things here and hope to get it done within this week.

**Comment by a-teammate** [comment]
So?

File: license.md [hunk 1]
```diff
@@ -36,50 +17,3 @@ freely, subject to the following restrictions:
...
-
-
-
-## Credits
-
-### Inexor
-
-* see wiki for the actual list of developers and content designers
-
-### Sauerbraten/Cube2
```
Comment (a-teammate) [inline license.md]: Yes, you can append to this pr if you want

**Comment by a-teammate** [comment]
kk thanks for the credits.md

**Pull request merged**